sandbox/urls.py

```diff
@@ -62,3 +62,8 @@
         + staticfiles_urlpatterns()
         + urlpatterns
     )
+
+handler400 = "richie.apps.core.views.error.error_400_view_handler"
+handler403 = "richie.apps.core.views.error.error_403_view_handler"
+handler404 = "richie.apps.core.views.error.error_404_view_handler"
+handler500 = "richie.apps.core.views.error.error_500_view_handler"
```

Comment:
I think it's worth be documented somewhere? In an integration guide?

Reply:
Hmm this is a django feature.

Reply:
True. But I wish there was an integration guide for beginners...

Reply:
You are right. This is one of the next big steps, I noted to address this aspect.
#1235